### PR TITLE
Build fix

### PR DESCRIPTION
**Pull request by enjeyw**
**Describe the Pull Request**

The new circle flow is failing the deploy step because a couple of containers weren't defined properly. Quick fix to get it working.
Also there's this weird case weird circle rebuilds occasionally during the backend tests, which causes everything to fail. I think it's due to the layer cache checksum changing for unknown reasons, but i haven't got enough data to confirm just yet.
**Todo**

- [ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
     build:
       context: .
       dockerfile: app/Dockerfile
-    image: server
+    image: "server:${GIT_HASH}"
```
Comment (enjeyw) [inline docker-compose.yml:7]: Circle is prone to pulling previous builds without this

File: .circleci/config.yml [hunk 2]
```diff
@@ -194,6 +196,7 @@ workflows:
             branches:
               only:
                 - master
+                - build-fix
```
Comment (michielderoos) [inline .circleci/config.yml:199]: Should remove this before merge IMO, but the rest looks good to me! (contingent on deploy working with migrations changed)

Comment (michielderoos) [reply .circleci/config.yml:199]: Update: Build seems to still be broken, an issue with docker-compose

Comment (michielderoos) [reply .circleci/config.yml:199]: Tentative approval once `docker-compose` gets up 🚀

**Pull request merged**